packages/instant/.production.discharge.json

```diff
@@ -0,0 +1,13 @@
+{
+    "domain": "instant.0xproject.com",
+    "build_command": "yarn build",
+    "upload_directory": "umd",
+    "index_key": "instant.js",
+    "error_key": "404.html",
+    "trailing_slashes": true,
+    "cache": 3600,
```

Comment:
Do we feel comfortable with this cache setting? Is there a way we could kill the cache if we had an urgent bug?

Reply:
discharge also does it when you deploy a new version

Reply:
ok, sweet